test/functional/rpc_packages.py

```diff
@@ -306,5 +306,80 @@ def test_rbf(self):
         }]
         self.assert_testres_equal(self.independent_txns_hex + [signed_replacement_tx["hex"]], testres_rbf_package)
 
+    def assert_equal_package_results(self, node, testres_package, submitres_package):
+        """Assert that a successful submitpackage result is consistent with testmempoolaccept
+        results and getmempoolentry info. Note that the result structs are different and, due to
+        policy differences between testmempoolaccept and submitpackage (i.e. package feerate),
+        some information may be different.
+        """
+        for testres_tx in testres_package:
+            # Grab this result from the submitres
+            submitres_tx = submitres_package["tx-results"][testres_tx["wtxid"]]
+            assert_equal(submitres_tx["txid"], testres_tx["txid"])
+            if "allowed" in testres_tx and testres_tx["allowed"]:
```

Comment:
Is this always true?

Reply:
Not if we have partial submissions - added that test case and an explanation comment